S3TransferUtilitySample/AndroidManifest.xml

```diff
@@ -5,16 +5,13 @@
 
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
-    
-    
-    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
-    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
```

Comment:
Huh! [RIP, storage permissions](https://stackoverflow.com/q/58433728/12442906): you did us well.